### PR TITLE
add new websites (to comply with the new api)

### PR DESCRIPTION
**Pull request by Brandonbr1**

### CODE REVIEW

File: lists/minecraft.yaml [hunk 1]
```diff
@@ -2014,3 +2014,39 @@
...
   reason: Illegal redistribution
+- domain: resourcepacks24.de
+  notes: /
+  path: /
+  reason: Illegal redistribution
+- domain: modminecraft.com
+  notes: /
+  path: /
+  reason: Illegal redistribution
+- domain: shadersmods.com
```
Comment (dada513) [inline lists/minecraft.yaml:2025]: does not redistribute, links to original

File: lists/minecraft.yaml [hunk 2]
```diff
@@ -2014,3 +2014,39 @@
...
+  reason: Illegal redistribution 
+- domain: www.mtmods.com
+  notes: /
+  path: /
+  reason: Illegal redistribution
+- domain: www.mc-mod.net 
+  notes: /
+  path: /
+  reason: Illegal redistribution
+- domain: download.komputerswiat.pl
```
Comment (dada513) [inline lists/minecraft.yaml:2049]: duplicate

File: lists/minecraft.yaml [hunk 3]
```diff
@@ -2014,3 +2014,39 @@
...
+  reason: Illegal redistribution
+- domain: worldofminecraft.com
+  notes: whole community that redustibutes minecraft mods
+  path: /
+  reason: Illegal redistribution 
+- domain: www.mtmods.com
+  notes: /
+  path: /
+  reason: Illegal redistribution
+- domain: www.mc-mod.net
```
Comment (dada513) [inline lists/minecraft.yaml:2045]: duplicate

File: lists/minecraft.yaml [hunk 4]
```diff
@@ -2014,3 +2014,39 @@
...
+  reason: Illegal redistribution
+- domain: mcpedl.com
+  notes: /
+  path: /
+  reason: Illegal redistribution
+- domain: worldofminecraft.com
+  notes: whole community that redustibutes minecraft mods
+  path: /
+  reason: Illegal redistribution 
+- domain: www.mtmods.com
```
Comment (dada513) [inline lists/minecraft.yaml:2041]: does not redistr

File: lists/minecraft.yaml [hunk 5]
```diff
@@ -2014,3 +2014,39 @@
...
+  reason: Illegal redistribution
+- domain: modminecraft.com
+  notes: /
+  path: /
+  reason: Illegal redistribution
+- domain: shadersmods.com
+  notes: /
+  path: /
+  reason: Illegal redistribution
+- domain: 99minecraft.com
```
Comment (dada513) [inline lists/minecraft.yaml:2029]: duplicate

File: lists/minecraft.yaml [hunk 6]
```diff
@@ -2014,3 +2014,39 @@
...
+  notes: /
+  path: /
+  reason: Illegal redistribution
+- domain: mcpedl.com
```
Comment (dada513) [inline lists/minecraft.yaml:2033]: im not sure about mcpedl, can you give your feedback on #426?

**Review by dada513 — CHANGES_REQUESTED** [review]
Also, list should be sorted alphabetically. Once you're done with my review I can sort it before merge

**Comment by Brandonbr1** [comment]
@dada513  did all the changes you asked  me to

**Comment by dada513** [comment]
> @dada513 did all the changes you asked me to

waiting for @berrysauce feedback on mcpdl

**Review by berrysauce — CHANGES_REQUESTED** [review]
Remove mcpedl.com for now. We need more time to review/discuss it.

**Pull request merged**